flow/layers/image_filter_layer.cc

```diff
@@ -49,7 +62,7 @@ void ImageFilterLayer::Paint(PaintContext& context) const {
   paint.setImageFilter(filter_);
 
   Layer::AutoSaveLayer save_layer =
-      Layer::AutoSaveLayer::Create(context, paint_bounds(), &paint);
+      Layer::AutoSaveLayer::Create(context, child_paint_bounds_, &paint);
```

Comment:
Nit: maybe add a comment here to explicitly explain why unfiltered `child_paint_bounds` is needed instead of `paint_bounds()`. It's not very intuitive (as we missed in the first time), and it's different from most call sites.

Reply:
How does the latest change look?